### PR TITLE
[range.cartesian.view] Fix neglect for parent_

### PR DESCRIPTION
**Pull request by hewillk**
This seems editorial bug in the paper.

### CODE REVIEW

**Comment by tkoeppe** [comment]
@jwakely Does this require an LWG issue?

**Comment by hewillk** [comment]
If this is the LWG, there is *no* way to implement a functional `cartesian_view` according to the draft until the LWG makes any decisions.

**Comment by tkoeppe** [comment]
> If this is the LWG, there is _no_ way to implement a functional `cartesian_view` according to the draft until the LWG makes any decisions.

That may well be so, but is not an editorial concern.

`vector` also wasn't implementable until C++23, but that also didn't get fixed editorially.

File: source/ranges.tex [hunk 1]
```diff
@@ -14654,14 +14660,15 @@
...
+constexpr explicit @\exposid{iterator}@(@\exposidnc{Parent}@& parent,
+  tuple<iterator_t<@\exposid{maybe-const}@<Const, First>>,
+        iterator_t<@\exposid{maybe-const}@<Const, Vs>>...> current);
 \end{itemdecl}
 
 \begin{itemdescr}
 \pnum
 \effects
-Initializes \exposid{current_} with \tcode{std::move(current)}.
+Initializes \exposid{parent_} with \tcode{addressof(parent)} and \exposid{current_} with \tcode{std::move(current)}.
```
Comment (jwakely) [inline source/ranges.tex:14671]:
```suggestion
Initializes \exposid{parent_} with \tcode{addressof(parent)}
and \exposid{current_} with \tcode{std::move(current)}.
```

File: source/ranges.tex [hunk 2]
```diff
@@ -14673,7 +14680,7 @@
 \begin{itemdescr}
 \pnum
 \effects
-Initializes \exposid{current_} with \tcode{std::move(i.\exposid{current_})}.
+Initializes \exposid{parent_} with \tcode{i.\exposid{parent_}} and \exposid{current_} with \tcode{std::move(i.\exposid{current_})}.
```
Comment (jwakely) [inline source/ranges.tex:14683]:
```suggestion
Initializes \exposid{parent_} with \tcode{i.\exposid{parent_}}
and \exposid{current_} with \tcode{std::move(i.\exposid{current_})}.
```

**Review by jwakely — CHANGES_REQUESTED** [review]
Use semantic line breaks:
https://github.com/cplusplus/draft/wiki/Specification-Style-Guidelines#latex-source-code-formatting

**Comment by jwakely** [comment]
> If this is the LWG, there is _no_ way to implement a functional `cartesian_view` according to the draft until the LWG makes any decisions.

That's fine. This is not a final standard, there is no great urgency to fix defects in a draft.

Implementors can easily do this without waiting for an LWG decision. It's common that we implement fixes before they make it into the draft.

Although the changes here seem correct, I don't think it's editorial. Please submit an LWG issue instead, thanks!

**Comment by tkoeppe** [comment]
Could you kindly edit the pull request to avoid sounding disparaging of the existing work and the people who have contributed it?

**Comment by hewillk** [comment]
@tkoeppe  Sorry for the misunderstanding caused by the misuse of words, I will be more careful with words in the future pull.

@jwakely, thanks for the suggestion.
The reason I insist this is an editorial fix is that it was clearly an oversight, and I believe the fix was what the author originally intended. Maybe we can ask the author again about this opinion after he is on vacation.

If this is definitely an LWG, I'd like to ask _when_ it would be appropriate to submit it to the LWG, as the C++23 draft hasn't been published yet.

**Comment by jensmaurer** [comment]
Specifically addressing your question, you should submit an issue to lwgchair now, so that the matter does not get lost.

**Comment by hewillk** [comment]
Closed and intends to submit LWG for this.

**Pull request closed without merging**